src/Provider/DataSourceProvider.js

```diff
@@ -69,7 +76,7 @@ export default {
                 const extDest = extentsDestination[i];
 
                 // Already fetched and parsed data that can be used
-                const validedParsedData = isValidData(parsedData[i], extDest, layer.isValidData) || source.parsedData;
+                const validedParsedData = !command.forceUpdate && (isValidData(parsedData[i], extDest, layer.isValidData) || source.parsedData);
```

Comment:
Maybe there's a duplicate in the verification, because `parsedData` could be deleted in `refreshMaterial`.